.github/workflows/release-on-milestone-closed.yml

```diff
@@ -21,27 +21,17 @@ jobs:
         with:
           command-name: "laminas:automatic-releases:release"
         env:
-          "GITHUB_TOKEN": ${{ secrets.GITHUB_TOKEN }}
+          "GITHUB_TOKEN": ${{ secrets.ORGANIZATION_ADMIN_TOKEN }}
           "SIGNING_SECRET_KEY": ${{ secrets.SIGNING_SECRET_KEY }}
           "GIT_AUTHOR_NAME": ${{ secrets.GIT_AUTHOR_NAME }}
           "GIT_AUTHOR_EMAIL": ${{ secrets.GIT_AUTHOR_EMAIL }}
 
-      - name: "Create Merge-Up Pull Request"
-        uses: "laminas/automatic-releases@1.0.1"
+      - name: "Create new milestones"
+        uses: "laminas/automatic-releases@v1"
```

Comment:
If the following is added to the dependabot config, dependabot will also update the github actions:

```yaml

  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: daily
      time: '04:00'
    open-pull-requests-limit: 10
    reviewers:
      - ...
    assignees:
      - ...
    labels:
      - "dependencies"
    commit-message:
      include: "scope"
      prefix: "github-actions"
```